Makefile

```diff
@@ -198,11 +198,13 @@ base.opt:
 
 COMPLIBDIR=$(LIBDIR)/compiler-libs
 
-INSTALL_BINDIR=$(DESTDIR)$(BINDIR)
-INSTALL_LIBDIR=$(DESTDIR)$(LIBDIR)
-INSTALL_COMPLIBDIR=$(DESTDIR)$(COMPLIBDIR)
-INSTALL_STUBLIBDIR=$(DESTDIR)$(STUBLIBDIR)
-INSTALL_MANDIR=$(DESTDIR)$(MANDIR)
+override shellquote = ""'''$(subst ','\'',$1)'#)'
+
+INSTALL_BINDIR:=$(call shellquote,$(DESTDIR)$(BINDIR))
```

Comment:
Why do we need to shellquote these variables and not directly DESTDIR, BINDIR, etc?
Why should it be done in the Makefile rather than the configure script?